migration/faqs.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
```

Comment:
I would suggest add a bit more context:
"# Frequently Asked Questions about Nimiq Migration PoS" 

This is because if we have in the future another page with the same heading, it will create problems when indexing elements for the search algorithm